azkaban-common/src/main/java/azkaban/scheduler/Schedule.java

```diff
@@ -224,6 +244,23 @@ private DateTime getNextRuntime(long scheduleTime, DateTimeZone timezone,
     return date;
   }
 
+  /**
+   *
+   * @param scheduleTime represents the time when Schedule Servlet receives the Cron Schedule API call.
```

Comment:
Why not just use time now since the delta between these two times should be very small.